exercises/binary/test/test_binary.c

```diff
@@ -0,0 +1,85 @@
+#include "vendor/unity.h"
+#include "../src/binary.h"
+
+void b1_is_decimal_1(void)
```

Comment:
Can you prefix these guys with `test_` to better indicate that they're the actual tests and not just helper functions?